packages/javascript/bh-shared-ui/src/utils/jobs.ts

```diff
@@ -13,24 +13,43 @@
 // limitations under the License.
 //
 // SPDX-License-Identifier: Apache-2.0
+
 import type { ScheduledJobDisplay } from 'js-client-library';
-import { DateTime, Interval } from 'luxon';
-import { OptionsObject } from 'notistack';
 import { StatusType } from '../components';
-import { LuxonFormat } from './datetime';
 
-export const JOB_STATUS_MAP: Record<number, { label: string; status: StatusType; pulse?: boolean }> = {
-    [-1]: { label: 'Invalid', status: 'bad' },
-    0: { label: 'Ready', status: 'good' },
-    1: { label: 'Running', status: 'pending', pulse: true },
-    2: { label: 'Complete', status: 'good' },
-    3: { label: 'Canceled', status: 'bad' },
-    4: { label: 'Timed Out', status: 'bad' },
-    5: { label: 'Failed', status: 'bad' },
-    6: { label: 'Ingesting', status: 'pending', pulse: true },
-    7: { label: 'Analyzing', status: 'pending' },
-    8: { label: 'Partially Completed', status: 'pending' },
-};
+export const JOB_STATUS_MAP: Record<number, string> = {
```

Comment:
Split the job status map into a bare lookup and a map specific to the status indicator.